Documentation/user-guides/getting-started.md

```diff
@@ -146,3 +146,13 @@ spec:
     matchLabels:
       team: frontend
 ```
+
+## What's Next
+
+- Learn about the [resoucres]({{<ref "design.md">}}) managed by Prometheus-Operator
+- API Reference for [ServiceMonitor](https://prometheus-operator.dev/docs/api-reference/api/#monitoring.coreos.com/v1.ServiceMonitor) and [PodMonitor](https://prometheus-operator.dev/docs/api-reference/api/#monitoring.coreos.com/v1.PodMonitor) CRD
+- Instructions on [deploying Prometheus and Alertmanager instances](https://prometheus-operator.dev/docs/platform/getting-started/)
+- Set up Alerting Rules
+- Learn about the [Prometheus Agent]({{<ref "prometheus-agent.md">}}) CRD
```

Comment:
Should we use the same blocks we have here? https://prometheus-operator.dev/docs/getting-started/introduction/
```suggestion
- API Reference for [ServiceMonitor](https://prometheus-operator.dev/docs/api-reference/api/#monitoring.coreos.com/v1.ServiceMonitor) and [PodMonitor](https://prometheus-operator.dev/docs/api-reference/api/#monitoring.coreos.com/v1.PodMonitor) CRD
```

I wouldn't put a ton of things here, I think people get lost when there are too many options.

And what do you think about using the same blocks used at the bottom of this page: https://prometheus-operator.dev/docs/getting-started/introduction/

Reply:
> Should we use the same blocks we have here? https://prometheus-operator.dev/docs/getting-started/introduction/

That's a great idea. :saluting_face: . We can name the "What's Next' section to 'Next Steps' to maintain similarity.

Reply:
> I wouldn't put a ton of things here, I think people get lost when there are too many options.

I think we need to discuss this more. If we only add one link then it would just be better to change the order of docs and follow the normal navigation we currently have. However, the aim of the 'What's Next' section is to give options to what the user wants to do. So, it would make sense to add the topics that are connected to the topics discussed on the current page.

We can remove Alerting Rules as it would be the next page(after it is added) in the navigation. Other than that, all things should be mentioned from my perspective.